nemesis/arena_prep.py

```diff
@@ -2187,64 +2056,64 @@ def t_plains(cap_num: int = 1, auto_haste: bool = False):
     memory.main.full_party_format(front_line="yuna", full_menu_close=False)
     menu.remove_all_nea()
     memory.main.close_menu()
-    prefArea = plains_next(end_goal=cap_num)
-    print("Next area: ", prefArea)
-    neEquip = False
+    pref_area = plains_next(end_goal=cap_num)
+    logger.debug(f"Next area: {pref_area}")
+    ne_equip = False
 
     checkpoint = 0
     while not memory.main.get_map() in [194, 374]:
         if memory.main.user_control():
             if memory.main.dodge_lightning(game_vars.get_l_strike()):
-                print("Strike!")
+                logger.debug("Strike!")
                 game_vars.set_l_strike(memory.main.l_strike_count())
-            if prefArea in [3, 4] and not neEquip:
+            if pref_area in [3, 4] and not ne_equip:
                 logger.debug(f"No Encounters armor on char: {game_vars.ne_armor()}")
                 menu.equip_armor(character=game_vars.ne_armor(), ability=0x801D)
-                neEquip = True
+                ne_equip = True
                 if checkpoint in [4, 5]:
                     checkpoint = 6
                 if checkpoint in [9, 10]:
                     checkpoint = 11
-            elif checkpoint in [8, 12] and prefArea in [3, 4]:
+            elif checkpoint in [8, 12] and pref_area in [3, 4]:
                 checkpoint = 20
-                print("Back to agency", checkpoint)
-            elif checkpoint in [6, 14, 15, 16] and prefArea == 1:
+                logger.debug(f"Back to agency{checkpoint}")
```

Comment:
probably want a space in here before {checkpoint}

Reply:
Updated, thanks.